gateway/api/views.py

```diff
@@ -347,6 +347,38 @@ def stop(self, request, pk=None):  # pylint: disable=invalid-name,unused-argumen
                         )
         return Response({"message": message})
 
+    @action(methods=["POST"], detail=True)
+    def add_runtimejob(
+        self, request, pk=None
+    ):  # pylint: disable=invalid-name,unused-argument
+        """Add RuntimeJob to job"""
+        tracer = trace.get_tracer("gateway.tracer")
+        ctx = TraceContextTextMapPropagator().extract(carrier=request.headers)
+        with tracer.start_as_current_span("gateway.job.add_runtimejob", context=ctx):
+            job = self.get_object()
+            runtimejob = RuntimeJob(
+                job=job,
+                runtime_job=request.data.get("runtime_job"),
```

Comment:
Should we check that `runtime_job` got passed in request somehow to display user friendly error? 

Something like
```python
if not request.data.get("runtime_job"):
    return Response({"message": "Got empty `runtime_job` field. Please, specify `runtime_job`."}, code / status=400)
```